packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -124,13 +124,15 @@ void main() {
 
     result = await binding.testExtension('debugDumpRenderTree', <String, String>{});
     expect(result, <String, String>{});
-    expect(console, <String>[
-      'RenderView\n'
-      '   debug mode enabled - ${Platform.operatingSystem}\n'
-      '   window size: Size(800.0, 600.0) (in physical pixels)\n'
-      '   device pixel ratio: 1.0 (physical pixels per logical pixel)\n'
-      '   configuration: Size(800.0, 600.0) at 1.0x (in logical pixels)\n'
-      '\n'
+    expect(console, <Matcher>[
+      matches(
+        'RenderView\\([0-9]+\\)\n'
+        '   debug mode enabled - ${Platform.operatingSystem}\n'
```

Comment:
There seems to be no API to escape it - I filed https://github.com/dart-lang/sdk/issues/28784